apps/remix-ide/src/app/tabs/compileTab/compilerContainer.js

```diff
@@ -201,8 +201,9 @@ class CompilerContainer {
       title="Estimated number of times each opcode of the deployed code will be executed across the life-time of the contract."
       onchange=${() => this.onchangeRuns()}
     >`
-    if (this.compileTabLogic.optimize) this._view.runs.removeAttribute('disabled')
-    else {
+    if (this.compileTabLogic.optimize) {
+      this._view.runs.value = this.compileTabLogic.runs
```

Comment:
i think you should still remove disable

Reply:
by checking it locally, that doesn't seem required. Because by default it is enabled and disabled only when `this.compileTabLogic.optimize === false
`

Reply:
probably because disable is removed here https://github.com/ethereum/remix-project/blob/master/apps/remix-ide/src/app/tabs/compileTab/compilerContainer.js#L347

Reply:
by default, it should be disabled. I would prefer to leave it as it is setting the initial state.

Reply:
> Please also add e2e test

will do once https://github.com/ethereum/remix-project/pull/623 is merged